pkg/nuctl/command/import.go

```diff
@@ -114,17 +118,46 @@ func (i *importCommandeer) importFunction(ctx context.Context, functionConfig *f
 func (i *importCommandeer) importFunctions(ctx context.Context,
 	functionConfigs map[string]*functionconfig.Config,
 	project *platform.ProjectConfig) error {
-	errGroup, _ := errgroup.WithContext(ctx, i.rootCommandeer.loggerInstance)
-
-	i.rootCommandeer.loggerInstance.DebugWithCtx(ctx, "Importing functions", "functions", functionConfigs)
+	importFailed := atomic.Bool{}
+	i.rootCommandeer.loggerInstance.DebugWithCtx(ctx,
+		"Importing functions",
+		"functions", functionConfigs)
+	wg := sync.WaitGroup{}
+	var sem = semaphore.NewWeighted(int64(runtime.NumCPU()))
```

Comment:
how about making it configurable `--concurrency` same as redeploying?
We can set the default to `runtime.NumCPU()`

Reply:
done!